gspread/auth.py

```diff
@@ -299,11 +317,15 @@ def service_account_from_dict(info, scopes=DEFAULT_SCOPES):
 
     :param info (Mapping[str, str]): The service account info in Google format
     :param list scopes: The scopes used to obtain authorization.
+    :type client_factory: :class:`gspread.ClientFactory`
+    :param client_factory: A factory function that returns a client class.
+        Defaults to :class:`gspread.Client` (but could also use
+        :class:`gspread.BackoffClient` to avoid rate limiting)
 
-    :rtype: :class:`gspread.Client`
+    :rtype: :class:`gspread.ClientFactory`
```

Comment:
and this is the last one :slightly_smiling_face: to update with `` :class:`gspread.client.Client` ``

Reply:
Changed!! 😁